pandas/tests/arrays/integer/test_arithmetic.py

```diff
@@ -317,3 +317,13 @@ def test_abs_nullable_int(any_signed_nullable_int_dtype, source, target):
     result = abs(s)
     expected = pd.array(target, dtype=dtype)
     tm.assert_extension_array_equal(result, expected)
+
+
+@pytest.mark.parametrize("op", ["__neg__", "__abs__"])
```

Comment:
can we also test \_\_invert__, since I believe has the same issue.

Reply:
Good catch, that required another change